Release Notes/2017/2017-07-18-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,27 @@
+{{{
+"title": "Cloud Platform - Release Notes: July 18 2017",
+"date": "7-18-2017",
+"author": "Ben Swoboda",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (2)
+
+* __Cloud Application Manager__
+
+  - __New Workspace Switcher__
+
+    With the new workspace switcher, users will now be able to:
+
+    * See the hierarchy of Organizations, CostCenters, and Workspaces.
+    * Mark a Workspace as a favorite
+    * See a list of favorite Workspaces
+
+ This will provide an easy way to navigate to frequently-visited Workspaces and a quick view of which Organization and Costcenter the Workspace belongs.
+
+* __IaaS__
```

Comment:
is there a reason we keep saying DE3 as its already been stated twice now.  separately, is there a reason we are not mentioning object storage germany is live?